src/index.js

```diff
@@ -53,10 +53,15 @@ export default (maybeElementId: YouTubePlayerType | HTMLElement | string, option
   options.events = YouTubePlayer.proxyEvents(emitter);
 
   const playerAPIReady = new Promise((resolve: (result: YouTubePlayerType) => void) => {
-    if (typeof maybeElementId === 'string' || maybeElementId instanceof HTMLElement) {
```

Comment:
As the description suggests, the `maybeElementId` can be either ID of an element or an existing YT.Player instance. This would restrict it to only the YT.Player instance.

If this change needs to be done for some compatibility, then the JSDoc comment needs to be updated and a comment needs to be added to explain the reason for the change.

Reply:
I'm afraid I don't understand.  As the jsdoc description says, maybeElementId can be a string elementId, an DOM element, or a YT.Player instance.  That functionality is supposed to remain unchanged by this PR and I believe it is because we're using this change in production from my fork, plus all the specs pass.

I'm attempting to fix a small corner case where you if you pass in a element that is inside an iframe or a popup window, the element is valid and a YT.Player can be instantiated inside it, however `typeof` will return undefined, and the original code would fail in the `illegal state` block.  This worked on an earlier version of YouTubePlayer but broke after we upgraded to the latest version.

FYI: the code that breaks is here:  
https://github.com/openstax/tutor-js/blob/master/tutor/src/screens/assignment-builder/footer/builder-popup.js#L55-L84

PopupWindow  opens a new window and renders it's children inside it:
https://github.com/openstax/tutor-js/blob/master/shared/src/components/popout-window.jsx#L114-L120

Reply:
Also, thanks for considering this change. While it's a tiny issue, I think addressing it will make YouTube Player even more bulletproof. 😁